plugin-base/src/main/java/io/everytrade/server/parser/exchange/BlockchainApiTransactionBean.java

```diff
@@ -51,6 +51,9 @@ public TransactionCluster toTransactionCluster(boolean isWithFee) {
         if (ParserUtils.equalsToZero(originalAmount)) {
             throw new IllegalArgumentException("Crypto amount can't be zero.");
         }
+        final boolean isWithFee =
```

Comment:
Where is isWithFee used???

Reply:
It is used few rows bellow in if:
        if (ParserUtils.equalsToZero(feeAmount) || isIgnoredFee || !isWithFee) {
            related = Collections.emptyList();